modules/replicator/src/main/java/org/apache/ignite/internal/replicator/message/ReplicaRequest.java

```diff
@@ -30,4 +32,20 @@ public interface ReplicaRequest extends NetworkMessage {
      * @return Replication group id.
      */
     String groupId();
+
+    /**
+     * Gets a primary replica.
+     *
+     * @return Gets a primary replica.
+     */
+    @Marshallable
+    ClusterNode primaryReplica();
+
+    /**
+     * Gets a raft term.
+     *
+     * @return Gets a raft term.
+     */
+    @Marshallable
+    Long term();
```

Comment:
Long is a simple type. That means the annotation (@Marshallable) does not require.

Reply:
Is the property (term) really required? I sure, we will use a replication without RAFT.

Reply:
Long is an object. So it need Marshallable.
I added TODO for "term".